docs/layers/choropleth.rst

```diff
@@ -88,6 +88,12 @@ The ``Choropleth`` layer is then created specifying on which key the colormap is
     )
 
 
+Other options for the colormap can be found by running ``linear`` in the notebook cell:
```

Comment:
Maybe we can also point to the branca documentation there: https://python-visualization.github.io/branca

Reply:
Good idea, I Added the link.